common/setups/rasr/hybrid_system.py

```diff
@@ -437,6 +372,8 @@ def nn_recognition(
         rtf: int,
         mem: int,
         epochs: Optional[List[int]] = None,
+        use_epoch_for_compile=False,
+        forward_output_layer="output",
```

Comment:
`forward_output_layer` is not automatically inferred from the `returnn_config` anymore. Are there cases where it does not match? Besides, the default is changed from `"log_output"` to `"output"`, is that desired?

Reply:
Log output was something I changed at some point, the original was `output`, same as with the dependency on the `returnn_config`. Now it is back like it was before.